bin/aurelia.js

```diff
@@ -56,17 +56,15 @@ function handleCommands(env) {
   }
 
   aurelia.init({
-    env : env
+      env : env
+    , argv: argv
   });
 
-  if (env.configPath) {
+  if (env.configPath && env.modulePath) {
     require(env.configPath);
```

Comment:
If we `require(env.configPath)`

The Aureliafile contains.

```
require('aurelia-cli');
```

This does not work. if the modulePath does not exist.